client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/rates-step/shipping-rate.js

```diff
@@ -19,45 +21,79 @@ class ShippingRate extends Component {
 	constructor() {
 		super();
 		this.state = {
-			signatureOption: false,
+			signatureOption: false
 		}
 	}
 
-	setSignatureOption = ( val ) => {
-		const {
-			rateObject: {
-				service_id,
-			},
-			updateValue,
-		} = this.props;
+	onSignatureChecked = ( isChecked, i, signatureOption ) => {
+		const { rateObject: { service_id }, updateValue } = this.props;
+		const selectedSignature = isChecked ? { id: i, value: signatureOption.netCost } : null;
+		this.setState( { selectedSignature } );
+		updateValue( service_id, signatureOption.value );
+	}
+
+	renderServices( carrier_id, signatureOptions, includedServices ) {
+
+		const servicesToRender = [];
 
-		this.setState( { signatureOption: val } );
-		updateValue( service_id, val );
+		if ( includedServices.tracking ) {
+			switch ( carrier_id ) {
+				case 'usps':
+					// Ideally this would come from connect-server, but we have no info from EasyPost API
```

Comment:
I think we could get rid of this comment now. Tracking will be coming from the connect-server now and we'll only show this on services that support it. I guess we can keep the switch statement as it'll say what kind of tracking it uses. It seems a little redundant to say USPS tracking as USPS offers no other kind. I think we could just say "Includes tracking"  and not use a switch statement.